[96m03-26-2021 15:41:51[0m | [96mproperties[0m - [96mgenheas.tools.properties[0m | [91mERROR[0m: [97m<class 'AttributeError'>[0m
[96m04-26-2021 15:06:48[0m | [96mtrain[0m - [96mgenheas.train[0m | [91mERROR[0m: [97mwhile parsing a block mapping
  in "./parameters.yml", line 6, column 1
expected <block end>, but found '<scalar>'
  in "./parameters.yml", line 47, column 25[0m
[96m04-26-2021 15:07:48[0m | [96mtrain[0m - [96mgenheas.train[0m | [91mERROR[0m: [97mwhile parsing a block mapping
  in "./parameters.yml", line 4, column 1
expected <block end>, but found '<scalar>'
  in "./parameters.yml", line 45, column 25[0m
[96m04-26-2021 15:09:27[0m | [96mtrain[0m - [96mgenheas.train[0m | [91mERROR[0m: [97mwhile parsing a block mapping
  in "./parameters.yml", line 4, column 1
expected <block end>, but found '<scalar>'
  in "./parameters.yml", line 45, column 25[0m
[96m04-26-2021 15:11:26[0m | [96mtrain[0m - [96m__main__[0m | [91mERROR[0m: [97m<oxidation_states> should match <elements_pool>[0m
[96m04-26-2021 15:53:27[0m | [96mgenerate[0m - [96m__main__[0m | [91mERROR[0m: [97mwhile scanning a simple key
  in "./parameters.yml", line 39, column 1
could not find expected ':'
  in "./parameters.yml", line 40, column 1[0m
[96m04-26-2021 15:53:59[0m | [96mgenerate[0m - [96m__main__[0m | [91mERROR[0m: [97mwhile scanning a quoted scalar
  in "./parameters.yml", line 39, column 14
found unexpected end of stream
  in "./parameters.yml", line 61, column 1[0m
[96m05-02-2021 03:23:39[0m | [96mtrain[0m - [96m__main__[0m | [91mERROR[0m: [97mwhile scanning a simple key
  in "./parameters.yml", line 38, column 1
could not find expected ':'
  in "./parameters.yml", line 40, column 1[0m
[96m05-28-2021 18:43:26[0m | [96mgenerate[0m - [96m__main__[0m | [91mERROR[0m: [97mwhile parsing a block mapping
  in "/home/ctetsass/Work/HEAs/Gen_HEAs/parameters.yml", line 4, column 1
expected <block end>, but found '<scalar>'
  in "/home/ctetsass/Work/HEAs/Gen_HEAs/parameters.yml", line 45, column 16[0m
[96m07-09-2021 20:20:10[0m | [96mtrain[0m - [96mgenheas.train[0m |[37m44[0m |[91mERROR[0m: [97mwhile parsing a flow sequence
  in "/home/ctetsass/Work/HEAs/Gen_HEAs/parameters.yml", line 31, column 17
expected ',' or ']', but got '<scalar>'
  in "/home/ctetsass/Work/HEAs/Gen_HEAs/parameters.yml", line 32, column 20[0m
[96m07-09-2021 20:21:22[0m | [96mtrain[0m - [96mgenheas.train[0m |[37m44[0m |[91mERROR[0m: [97mwhile parsing a flow sequence
  in "/home/ctetsass/Work/HEAs/Gen_HEAs/parameters.yml", line 30, column 17
expected ',' or ']', but got '<scalar>'
  in "/home/ctetsass/Work/HEAs/Gen_HEAs/parameters.yml", line 31, column 20[0m
[96m07-20-2021 16:40:12[0m | [96mproperties[0m - [96mgenheas.tools.properties[0m |[37m331[0m |[91mERROR[0m: [97m<class 'AttributeError'>[0m
[96m07-20-2021 16:42:01[0m | [96mproperties[0m - [96mgenheas.tools.properties[0m |[37m331[0m |[91mERROR[0m: [97m<class 'AttributeError'>[0m
[96m07-20-2021 16:44:10[0m | [96mproperties[0m - [96mgenheas.tools.properties[0m |[37m331[0m |[91mERROR[0m: [97m<class 'AttributeError'>[0m
[96m07-20-2021 16:45:47[0m | [96mproperties[0m - [96mgenheas.tools.properties[0m |[37m331[0m |[91mERROR[0m: [97m<class 'AttributeError'>[0m
[96m07-20-2021 16:46:39[0m | [96mproperties[0m - [96mgenheas.tools.properties[0m |[37m331[0m |[91mERROR[0m: [97m<class 'AttributeError'>[0m
[96m07-20-2021 16:48:22[0m | [96mproperties[0m - [96mgenheas.tools.properties[0m |[37m331[0m |[91mERROR[0m: [97m<class 'AttributeError'>[0m
[96m07-20-2021 16:51:57[0m | [96mproperties[0m - [96mgenheas.tools.properties[0m |[37m331[0m |[91mERROR[0m: [97m<class 'AttributeError'>[0m
[96m07-20-2021 16:56:09[0m | [96mproperties[0m - [96mgenheas.tools.properties[0m |[37m331[0m |[91mERROR[0m: [97m<class 'AttributeError'>[0m
[96m07-20-2021 16:57:20[0m | [96mproperties[0m - [96mgenheas.tools.properties[0m |[37m331[0m |[91mERROR[0m: [97m<class 'AttributeError'>[0m
[96m10-12-2021 15:57:31[0m | [96mtrain[0m - [96mgenheas.train[0m |[37m43[0m |[91mERROR[0m: [97mwhile scanning a simple key
  in "/home/ctetsass/python/genheas/genheas/parameters.yml", line 23, column 1
could not find expected ':'
  in "/home/ctetsass/python/genheas/genheas/parameters.yml", line 25, column 1[0m
[96m10-12-2021 18:55:29[0m | [96mgenerate[0m - [96m__main__[0m |[37m94[0m |[91mERROR[0m: [97mA load persistent id instruction was encountered,
but no persistent_load function was specified.[0m
[96m11-29-2021 18:00:49[0m | [96mtrain[0m - [96mgenheas.train[0m |[37m43[0m |[91mERROR[0m: [97mwhile scanning a simple key
  in "/home/ctetsass/python/genheas/genheas/parameters.yml", line 52, column 1
could not find expected ':'
  in "/home/ctetsass/python/genheas/genheas/parameters.yml", line 53, column 1[0m
[96m11-29-2021 19:19:35[0m | [96mgenerate[0m - [96mgenheas.generate[0m |[37m59[0m |[91mERROR[0m: [97mInvalid magic number; corrupt file?[0m
[96m11-29-2021 19:20:35[0m | [96mgenerate[0m - [96m__main__[0m |[37m59[0m |[91mERROR[0m: [97m[Errno 2] No such file or directory: 'Fe_Co_Ni_Zn_Ga/generations_750/best_model_0.1a-16p-4n.pth.tar'[0m
[96m11-29-2021 19:20:53[0m | [96mgenerate[0m - [96m__main__[0m |[37m59[0m |[91mERROR[0m: [97mError(s) in loading state_dict for Feedforward:
	size mismatch for l1.weight: copying a param with shape torch.Size([5, 110]) from checkpoint, the shape in current model is torch.Size([64, 32]).
	size mismatch for l1.bias: copying a param with shape torch.Size([5]) from checkpoint, the shape in current model is torch.Size([64]).[0m
[96m11-29-2021 22:18:59[0m | [96mgenerate[0m - [96mgenheas.generate[0m |[37m61[0m |[91mERROR[0m: [97m'nb_policies'[0m
[96m11-29-2021 22:21:46[0m | [96mgenerate[0m - [96mgenheas.generate[0m |[37m57[0m |[91mERROR[0m: [97m'policy_0-nn_0'[0m
[96m11-29-2021 22:22:06[0m | [96mgenerate[0m - [96mgenheas.generate[0m |[37m57[0m |[91mERROR[0m: [97m'_IncompatibleKeys' object has no attribute 'to'[0m
[96m11-29-2021 22:22:44[0m | [96mgenerate[0m - [96mgenheas.generate[0m |[37m58[0m |[91mERROR[0m: [97m'_IncompatibleKeys' object has no attribute 'to'[0m
